src/plugins/intel_gpu/tests/functional/remote_blob_tests/gpu_remote_tensor_tests.cpp

```diff
@@ -329,6 +341,280 @@ TEST_P(OVRemoteTensorInputBlob_Test, smoke_canInputRemoteTensor) {
     }
 }
 
+TEST_P(OVRemoteTensorInputBlob_Test, smoke_canInputOutputRemoteTensor) {
```

Comment:
As for 1) maybe we will throw error "Cannot call setShape for Blobs created on top of preallocated memory if shape was increased" but it's not very informative in that situation?

Reply:
1) In my thought, it sounds better to alert that the allocated remote tensor is not enough for the current shape.
Even though we allocate new memory, unfreed original device mem may cause memory leak. 

2) But I think it will be an issue for users to decide allocate enough memory .. Maybe we need a new hint for transformer to calculate it automatically for transformer models even in a very limited scenario. For example if we assume a scenario w/ optimum, then we can add such hints for well-known models ? Or.. we may provide users  estimated memory by bounded dynamic shape after compile_model done.